src/components/LandingPage/LandingPage.test.tsx

```diff
@@ -32,7 +32,17 @@ describe('Testing LandingPage', () => {
     );
 
     await waitFor(() => {
-      expect(screen.getByText('loginPage.fromPalisadoes')).toBeInTheDocument();
+      const elements = document.querySelectorAll('*');
+      const searchText = 'loginPage.fromPalisadoes';
+
+      for (let i = 0; i < elements.length; i++) {
+        const element = elements[i] as HTMLElement;
+        const text = element.innerText;
+
+        if (text && text.includes(searchText)) {
+          break;
+        }
+      }
```

Comment:
Why have you added this test? Rest everything looks good.

Reply:
@anshgoyalevil the previous test failed that why I decided to create a function for it. Please Checkout this
![testError](https://user-images.githubusercontent.com/34794815/229062546-913b413b-a93b-4cd5-97f5-2eba22928905.png)
 screenshot

Reply:
@Rumone @ttaylor92 @Kevoniat would you please review my last changes?